dataclients/kubernetes/definitions/routegroups.go

```diff
@@ -1,73 +1,71 @@
-package kubernetes
+package definitions
 
 import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"time"
 
 	"github.com/zalando/skipper/eskip"
 	"github.com/zalando/skipper/loadbalancer"
 )
 
-type routeGroupList struct {
-	Items []*routeGroupItem `json:"items"`
-}
+// adding Kubernetes specific backend types here. To be discussed.
+// The main reason is to differentiate between service and external, in a way
+// where we can also use the current global option to decide whether the service
+// should then be converted to LB. Or shall we expect the route group already
+// contain the pod endpoints, and ignore the global option for skipper?
+// --> As CRD we have to lookup endpoints ourselves, maybe via kube.go
```

Comment:
in case this is a new comment: yes we need this and users can not declare endpoints themselves without creating their own control loop, which is not what we want.

Reply:
this was only moved